app/helpers/listings_helper.rb

```diff
@@ -115,4 +117,31 @@ def price_quantity_per_unit(listing)
       ""
     end
   end
+
+  def shape_name(listing)
+    get_translations([listing.shape_name_tr_key], listing.communities.first).first
+  end
+
+  def action_button_label(listing)
+    get_translations([listing.action_button_tr_key], listing.communities.first).first
+  end
+
+  # private
+  def get_translations(translation_keys, community)
+    locale = I18n.locale.to_s
+    fallback = community.default_locale.to_s
+
+    result = TranslationService::API::Api.translations
+      .get(community.id, {
```

Comment:
This should probably be discussed through. Is the translations api as it should be?